#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-logfile.dpatch by Jens Schmalzing <jensen@debian.org>
##
## DP: Adjusts permission of the logfile.

@DPATCH@

diff -Naur pxe-1.4.2.orig/logfile.cc pxe-1.4.2/logfile.cc
--- pxe-1.4.2.orig/logfile.cc	2003-02-02 12:33:32.000000000 +0000
+++ pxe-1.4.2/logfile.cc	2006-06-30 10:07:43.000000000 +0000
@@ -57,7 +57,7 @@
 LogFile::Open(const char *filename)
 {
 	/* open the file */
-	umask(077);
+	umask(022);
 	logfile = new std::fstream(filename, std::ios::out|std::ios::app);
 	if(logfile == NULL)
 	{
